### DISCUSSION TITLE
Is it possible to change the port number of the *arr LXCs?

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#2004] [Q&A]
I'd like to use my Adguard to rewrite DNS so that I can access via a dns like app.home 
Adguard can't pass a port so I need to change the default LXC ports for the apps to be port 80 (if I understand correctly).
Is this possible?

**Comment by afmotta** [comment]
Use Nginx Proxy manager together with Adguard.
Using Adguard you point your local DNS to the Proxy Manager.
In Proxy Manager you set up the Proxy te redirect correctly the traffic to the IP:PORT needed.
